new/LIT7072QeneMamhera.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7072QeneMamhera" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">">መምህረ፡ ዝንቱ፡ ዓለም፡ አውስጣቴዎስ፡ ደብረ፡ አርማንያ፡ ወልደከ፡ በጽሐ፡ ለዘምሮ፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Mamǝhǝra zǝntu ʿālam, ʾAwǝsṭātewos, dabra ʾArmānyā waldaka baṣḥa la-zammǝro… (Qǝne of unknown type)</title>
```

Comment:
There is no special punctuation marks in the line, and we do not use any, therefore.  Three dots only in translation (as in other qene records).

Reply:
Do you mean I should change to a word divider at the end?

Reply:
No, they are fine and mark the verses; i meant two commas in English translation.

Reply:
Ah, ok, I understand. Thanks!

Reply:
There is a multiplicity of ways one can constract his/her own understanding of verses. It is better to reduce our interpretation exclusively to the translation :)